### PR TITLE
Add Chia Blockchain to TrueNAS CORE index

### PR DESCRIPTION
**Pull request by kmoore134**
Thank you for your submission to the FreeNAS community plugins repository!

Please ensure the following guidelines are met when submitting a new Plugin.

1. Add the entry to INDEX in alphabetical order
2. Includes a new icon in the ./icon/ directory
3. Ensure the submitted icon file is a valid PNG that is 128x128 in size
4. Add new plugin cirrus task

### CODE REVIEW

**Comment by fulder** [comment]
@kmoore134 If you would like to enable the automatic installation tests for this new plugin you can add a new chia task block for your new plugin to the  `.cirrus.yml` file here as well :)

```yaml
chia_task:
  <<: *INSTALL_PLUGIN
  only_if: "changesInclude('chia.json', '.cirrus/install_script.sh')"
  matrix:
    - freebsd_instance:
        image_family: freebsd-12-2
  env:
    PLUGIN_FILE: "chia.json"
```

File: chia.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    "plugin_schema": "2",
+    "release": "12.2-RELEASE",
+    "artifact": "https://github.com/kmoore134/iocage-plugin-chia.git",
+    "official": false,
+    "properties": {
+        "nat": 1,
+        "nat_forwards": "tcp(8444:8444)"
+    },
+    "pkgs": [
+        "lang/gcc9",
```
Comment (sonicaj) [inline chia.json:12]: Can we please format the list of package(s) correctly ?

**Review by sonicaj — APPROVED** [review]
LGTM, just one minor nit

**Comment by kmoore134** [comment]
> @kmoore134 If you would like to enable the automatic installation tests for this new plugin you can add a new chia task block for your new plugin to the `.cirrus.yml` file here as well :)
> 
> ```yaml
> chia_task:
>   <<: *INSTALL_PLUGIN
>   only_if: "changesInclude('chia.json', '.cirrus/install_script.sh')"
>   matrix:
>     - freebsd_instance:
>         image_family: freebsd-12-2
>   env:
>     PLUGIN_FILE: "chia.json"
> ```

Thanks for pointing that out, missed that one :)

**Comment by fulder** [comment]
@kmoore134 it looks like the new task failed to install the plugin during the commit just before the merge :cold_sweat:. 

I created a small PR in the plugin repo to fix the issue: https://github.com/kmoore134/iocage-plugin-chia/pull/1 looks like both the tests and manual installation on my TN CORE machine worked fine after that.

**Pull request merged**